### PR TITLE
LPD-38680 Ensure tree max height will not be exceeded

### PR DESCRIPTION
**Pull request by MarinhoFeliphe**
Related: https://liferay.atlassian.net/browse/LPD-38680

### CODE REVIEW

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:test:object

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java [hunk 1]
```diff
@@ -762,6 +762,84 @@ public void testBindDraftObjectDefinitions() throws Exception {
...
+			ObjectRelationshipEdgeException.class,
+			"Object relationship cannot be an edge because the tree max " +
+				"height will be exceeded",
+			() -> TreeTestUtil.bind(
+				_objectRelationshipLocalService,
+				Arrays.asList(objectRelationship)));
+	}
+
+	@Test
+	public void testBindObjectDefinitionTreesExceedingTreeMaxHeight()
```
Comment (marcelabc) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java:801]: Rename it to `testBindObjectDefinitionTreesExceedTreeMaxHeight()`

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

**Comment by pedro-oliveira446** [comment]
Resent : https://github.com/liferay-bpm/liferay-portal/pull/3727

**Pull request closed without merging**